### DISCUSSION TITLE
Simple Background Color

### DISCUSSION THREAD

**Discussion by ericmager** [gorhom/react-native-bottom-sheet#980] [Q&A] [answered]
I've tried for a few hours now and can't figure out how to change the sheet color. I don't need it to transition based on snapPoint or anything. I just want it always to be black instead of white. I've tried working with background component but this makes it transparent:
`  const renderBackdrop = useCallback(
    (props) => (
      <BottomSheetBackdrop {...props} style={{ backgroundColor: "#222222" }} />
    ),
    []
  );`

`      <BottomSheet
        enablePanDownToClose={true}
        ref={sheetRef}
        snapPoints={[150, 400]}
        borderRadius={10}
        backgroundComponent={renderBackdrop}
      >
        <SignUp />
      </BottomSheet>`

**Comment by giannifm** [comment, marked as answer]
Try using `backgroundColor` style on the `backgroundStyle` prop
Ex:
```
<BottomSheet
   ...
   backgroundStyle={{
      backgroundColor: "#000"
   }}>
</BottomSheet>
```

**Reply by ericmager** [reply]
thank you!
